views/administration/school.hbs

```diff
@@ -407,7 +408,7 @@
 						{{#ifConfig "FEATURE_VIDEOCONFERENCE_ENABLED" true}}
 						<div class="form-group">
 							<label>
-								<input type="checkbox" name="videoconference" value="true"
+								<input type="checkbox" name="videoconference" value="true" data-testid="school-admin-video-conference-checkbox"
```

Comment:
see above, either: `video-conference-checkbox` or `school-administration-video-conference-checkbox` or simply `administration-video-conference-checkbox`

Reply:
here updated the name to -->  school-administration-video-conference-checkbox